### PR TITLE
(#348) CMS Site Section tests 

### LINKED ISSUE #348: Site section - reassigning a landing page's site section
## As a [role], I want [feature], so that [benefit]

> **ESTIMATE** TBD

### Acceptance Criteria

```gherkin
    Scenario: Changing a site section of the landing page
        Given user is navigating to "/user/login"
        When user enters credentials
        And user clicks "Log in" button
        Then user is logged in and the user name "admin" is displayed in the toolbar
        And the tool bar appears at the top
        When user clicks on "Content" tab
        And user clicks on title with url "article-to-test-site-section" from the list of content
        And user clicks on the tool bar status green button "Published"
        And user clicks "View in edit form" button from other actions
        And user removes site section
        When user selects "/about-cancer/coping/feelings" site section
        And user clears out "Pretty URL" field
        And user fills out the following fields
            | Pretty URL | purl | field_pretty_url |
        When user saves the content page
        And user clicks on the tool bar status green button "Editing"
        And user selects "Quick Publish" from workflow actions

 Scenario: Verify that the test site section points to a new url after changing landing's page site section
        Given user is navigating to the front end site with path "/about-cancer/coping/feelings/purl"
        Then the current left navigation label has url "/about-cancer/coping/feelings"
        And page title is "Article to test Site Section"
        And left navigation label "Test Site Section" has url "/about-cancer/coping/feelings/purl"
```

### Resources:
* Feature: JIRA Ticket #
* Mockups: *Here goes a URL to or the name of the mockup(s) in inVision*
* Web Page: *Here goes a URL to an existing web page*

### Notes

*Some complementary notes if necessary:*

* > Here goes a quote from an email
* Here goes whatever useful information can exist…

---

### Prerequisites
- *required dependency for identifying the problem, scope and then solution for this issue*
- [ ] Short Spike Description - Issue #9999

### Tasks
- [ ] Short Task Description - Issue #9999
- [ ] Short Task Description - Issue #9999

### Technical Notes
- More notes, or technical nuances/direction

### PR DESCRIPTION
**Pull request by alinai**
Closes #348 

Updated the $SiteSections.feature file with test scenario of #348

### CODE REVIEW

**Comment by alinai** [comment]
cypress run on test-acsf:
image

cypress test run on dev-acsf:
image

**Comment by alinai** [comment]
cypress test run on test-ac:
image

File: cypress/e2e/CMS/$SiteSections/index.js [hunk 1]
```diff
@@ -122,4 +122,12 @@ And('{string} appears in position {int} in the side menu tree', (label, position
 
 And('user clicks on title with the url {string} from the list of content', (contentHref) => {
     cy.get(`a[href='${contentHref}']`).click();
+});
+
+Then('the current left navigation label has url {string}', (currentHref) => {
+    cy.get('ul.usa-sidenav__sublist li').find(`a[href='${currentHref}']`).should('be.visible');
```
Comment (KateMashkinaNIH) [inline cypress/e2e/CMS/$SiteSections/index.js:128]: this does not verify its a current page
try this instead cy.get('ul.usa-sidenav__sublist li a[aria-current="page"]').should('have.href',currentHref)

**Review by KateMashkinaNIH — CHANGES_REQUESTED** [review]

File: cypress/e2e/CMS/$SiteSections/index.js (hunk 1 above)
Comment (alinai) [reply cypress/e2e/CMS/$SiteSections/index.js:128]: Getting the following error: 

image

**Comment by alinai** [comment]
cypress test run on test-ac:
image

**Comment by alinai** [comment]
cypress run on dev-acsf:
image

**Comment by KateMashkinaNIH** [comment]
commented out drag and drop for now - i will re-open the ticket to investigate it later

**Pull request merged**